### ISSUE TITLE
Recommendations skip link needs to be within a landmark region

### ISSUE THREAD

**Issue by rhenshaw56** [bbc/simorgh#7479]
**Describe the bug**
At the moment, the skip link for the recommendations container sits outside the `section` element which is land marked for the recommendation component an this might be misleading to users as it will be interpreted as being the start of recommendations.

**To Reproduce**

1. Go to 'http://localhost:7080/mundo/23263889'
2. Inspect the recommendations component
3. See that the Skip anchor is placed outside the `section` element.
![Screen Shot 2020-08-12 at 17 06 23](https://user-images.githubusercontent.com/25608386/90039451-8e72b300-dcbe-11ea-98c7-1b1beea2cbda.png)

**Expected behavior**
The Skip link container should be within the `section` element which signifies the beginning of recommendations as everything inside it is a standalone.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [X] This bug fix is expected to need manual testing.

**Additional context**

**Comment by paruchurisilpa** [comment]
Testing:

URL -  'http://localhost:7080/mundo/23263889'
<img width="628" alt="Screenshot 2020-08-25 at 09 36 42" src="https://user-images.githubusercontent.com/1379277/91152278-aed74000-e6b6-11ea-9e0c-d9b18c3cee29.png">
